### ISSUE TITLE
Client side does not evaluate correctly

### ISSUE THREAD

**Issue by nossebro** [BetaPictoris/modrinth.py#3]
https://github.com/BetaPictoris/modrinth.py/blob/cd30f02577b23a8a50176a9d4512f0f01b98d662/modrinth/__init__.py#L112

should be: 

```Python
if self.clientSide == 'optional' or self.clientSide == 'required': 
```

**Comment by BetaPictoris** [comment]
Hello @nossebro, thank you for submitting this issue! It has been fixed in 692b099501c8ca2855cdbca836b43586c4cd495c and will be included in patch v0.1.3.

**Comment by BetaPictoris** [comment]
Patch v0.1.3b is now released and on [PyPi](https://pypi.org/project/modrinth/0.1.3/) and [GitHub](https://github.com/BetaPictoris/modrinth.py/releases/tag/v0.1.3b). Closing this issue...
